usn/usn.py

```diff
@@ -0,0 +1,129 @@
+import argparse
```

Comment:
Should we include the license template here as well?

Reply:
I will update it.

To be honest, I have no thought on this. I moved the license declaration to `LICENSE` file and simply thought it is a default license across the whole project. Please let me know if it is not a conventional way as far as you know. The project `ubuntu-server-netboot` has few number of files so it is still easy to add license to each file, so I will do that :)